lib/wasi/src/syscalls/wasix/callback_thread_local_destroy.rs

```diff
@@ -0,0 +1,35 @@
+use super::*;
+use crate::syscalls::*;
+
+// FIXME: remove , since it's no longer used.
```

Comment:
So this need to be removed? Shall I do it?

Reply:
Yes, please go ahead

Reply:
Just like the other callbacks, we need this so libc and the rust std library can tell the runtime what exported function to call when a thread is terminated and needs to cleanup local thread storage

Reply:
I have removed the FIXME comments.